src/actions.js

```diff
@@ -143,7 +146,34 @@ const getTaxesTypes = (summitId) => async (dispatch, getState, { apiBaseUrl, get
             `${apiBaseUrl}/api/v1/summits/${summitId}/tax-types`,
             customErrorHandler
         )(params)(dispatch).then((res) => {
-          return res;
+            return res;
+        }).catch((error) => {
+            return Promise.reject(error);
+        })
+    }
+    catch (e) {
+        console.log(e);
+        return Promise.reject(e);
+    }
+}
+
+export const getTicketDiscount = (promoCode) => async (dispatch, getState, { apiBaseUrl, getAccessToken }) => {
```

Comment:
this is a code duplicaton
u should refactor https://github.com/fntechgit/summit-registration-lite/blob/be2186c59eb1d96523b505598b5cea104ee2fef6/src/actions.js#L106 and use it